### PR TITLE
Change with_setting to accept any JSON value

### LINKED ISSUE #583: Playground settings can only be &str
## Expected Behavior

Playground settings should be able to accept any JSON type (although maybe not Object or Array?).

## Actual Behavior

[`with_setting`](https://docs.rs/async-graphql/2.9.9/async_graphql/http/struct.GraphQLPlaygroundConfig.html#method.with_setting) can only accept a `&str` for the value, so any values passed are serialised into a JSON string. These then stay as a string while being inserted into the playground settings.

```rust 
playground_source(
      GraphQLPlaygroundConfig::new("/admin/api/graphql")
          .with_setting("schema.polling.enable", "false"),
)
```

image

## Specifications

- Version: 2.9.9
- Platform: Linux

I think this could be solved by changing `GraphQLPlaygroundConfig`'s settings Hashmap into `Hashmap<&'str, Value>` and then changing `with_setting` to accept a Value. 

I'd be happy to put a PR together for this, let me know if you think this is viable.

### PR DESCRIPTION
**Pull request by tim-maddison-rnf**
Closes #583

### CODE REVIEW

**Comment by tim-maddison-rnf** [comment]
I haven't added anything to the changelog, let me know if you want me to do it.

**Comment by sunli829** [comment]
> I haven't added anything to the changelog, let me know if you want me to do it.

Yes, please do it. 😁

File: src/http/playground_source.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 use std::collections::HashMap;
 
+use crate::Value;
```
Comment (sunli829) [inline src/http/playground_source.rs:3]: Please modify it as follows: 😁

```rust
use std::collections::HashMap;

use serde::Serialize;

use crate::Value;
```

**Comment by sunli829** [comment]
Thank you!👏

**Comment by tim-maddison-rnf** [comment]
And thank you for working on async-graphql, I've only been using it for about a week, but it's been an absolute pleasure to work with.

**Pull request merged**